cmd/rpcdaemon/commands/zkevm_counters.go

```diff
@@ -149,7 +149,7 @@ func (zkapi *ZkEvmAPIImpl) EstimateCounters(ctx context.Context, rpcTx *zkevmRPC
 	txCounters := vm.NewTransactionCounter(tx, int(smtDepth), uint16(forkId), zkapi.config.Zk.VirtualCountersSmtReduction, false)
 	batchCounters := vm.NewBatchCounterCollector(int(smtDepth), uint16(forkId), zkapi.config.Zk.VirtualCountersSmtReduction, false)
 
-	_, err = batchCounters.AddNewTransactionCounters(txCounters)
+	_, err = batchCounters.AddNewTransactionCounters(nil, txCounters)
```

Comment:
There are quite a few methods that now take the existing counters as a parameter.  Would it be possible to add the existing counters to the constructor of the batchCounters itself to save affecting the methods?